tests/Integration/CheckCommandTest.php

```diff
@@ -75,3 +75,39 @@
 
     expect(trim($output))->toContain('INFO  Configuration file already exists.');
 });
+
+it('may fail with text option', function (): void {
+    $application = new Application;
+
+    $application->add(new CheckCommand);
+
+    $command = $application->find('check');
+
+    $commandTester = new CommandTester($command);
+
+    $commandTester->execute([
+        '--text' => 'This is a test with a typoo.',
+    ]);
+
+    $output = $commandTester->getDisplay();
+
+    expect(trim($output))->toContain('Did you mean: typo, typos, type, topi');
+});
+
+it('may pass with text option', function (): void {
+    $application = new Application;
+
+    $application->add(new CheckCommand);
+
+    $command = $application->find('check');
+
+    $commandTester = new CommandTester($command);
+
+    $commandTester->execute([
+        '--text' => 'This is a test without any typos.',
+    ]);
+
+    $output = $commandTester->getDisplay();
+
+    expect(trim($output))->toContain('PASS  No misspellings found in the given text.');
```

Comment:
Here we can do the same:

```suggestion
    expect(trim($output))->toContain('PASS  No misspellings found in the given text.')
        ->and($commandTester->getStatusCode())->toBe(0);
```